bin/setup-popsyn

```diff
@@ -0,0 +1,180 @@
+        text = text + \
+    f'''#SBATCH --mail-type=FAIL
+#SBATCH --mail-user={email}
+'''
+
+    text = text + \
+    f'''export PATH_TO_POSYDON={path_to_posydon}
+export PATH_TO_POSYDON_DATA={path_to_posydon_data}
+python ./run_metallicity.py {metallicity}
+'''
+    filename = f"slurm_array_{str_met}_Zsun.sl"
+    with open(filename, mode='w') as file:
+        file.write(text)
+
+def create_slurm_merge(metallicity, partition, email, merge_walltime, account, path_to_posydon, path_to_posydon_data):
+
+    str_met = convert_metallicity_to_string(metallicity)
+    if partition=='debug-cpu':
+        merge_walltime='00:14:00'
+
+    text = f'''#!/bin/bash
+#SBATCH --job-name=merge_pop_syn_{str_met}
+#SBATCH --output=./{str_met}_logs/popsyn_merge_{str_met}.out
+#SBATCH --mem-per-cpu=4G
+#SBATCH --time={merge_walltime}
+'''
+    if account != None:
+        text += f'#SBATCH --account={account}\n'
+
+    if partition != None:
+        text += f'#SBATCH --partition={partition}\n'
+    if email != None:
+        text = text + \
+        f'''#SBATCH --mail-type=FAIL
+#SBATCH --mail-user={email}
+'''
+    text = text + \
+        f'''export PATH_TO_POSYDON={path_to_posydon}
+export PATH_TO_POSYDON_DATA={path_to_posydon_data}
+python ./merge_metallicity.py {metallicity}
+'''
+    filename = f'slurm_merge_{str_met}_Zsun.sl'
+    with open(filename, mode='w') as file:
+        file.write(text)
+
+
+#################
+# MAIN FUNCTION
+#################
+
+if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('ini_file', help='a population ini file for which you want to setup a population run', type=str)
+    parser.add_argument('-j','--job_array', help='the number of job in the job array you want to run', type=int, default=100)
+    parser.add_argument('--email', help='an email for the slurm scripts', default=None)
+    parser.add_argument('--partition', help='what cluster partition you want to run the script on', default=None)
+    parser.add_argument('--walltime', help='the walltime you would like to use in SLURM format', default='23:00:00')
+    parser.add_argument('--merge_walltime', help='the walltime you would like to use for the merge script in SLURM format', default='12:00:00')
+    parser.add_argument('--account', help='the account you would like to use', default=None)
+    args = parser.parse_args()
+
+    synpop_params = binarypop_kwargs_from_ini(args.ini_file)    
+    metallicities = synpop_params['metallicity']
+    if synpop_params['number_of_binaries'] / args.job_array < 1:
+        print("The number of binaries is less than the job array length. Please increase the number of binaries or decrease the job array length")
+        exit()
+
+    create_run_script(args.ini_file)
+    create_merge_script(args.ini_file)
+    
+    print("Created run script") 
+    for MET in metallicities:
+        str_met = convert_metallicity_to_string(MET)
+        try:
+            os.mkdir(f'{str_met}_logs')
```

Comment:
I was wondering, whether we should better revert the naming here: first "logs" and then the metallicity, may even a directory for each metallicity withing the logs.

Reply:
If this is a wanted change, I suggest to implement this as a separate PR.
Currently, I've followed the convention of the batch folder names and BinaryPopulation filenames.